public/ng-views/maps.html

```diff
@@ -550,78 +621,109 @@ <h3 class="noTopMargin noBottomMargin">
                   ng-click="updateOffPeakVal(scenario[tabnav].peak.min, tabnav);
                             updateRouteScorecard(scenario[tabnav].routeId, tabnav)"/>
         </div>
         <div class="col-xs-6">
-          <input type="range" min="1" max="30"
+          <input type="range" min="1" max="30" value="10"
                   ng-model="scenario[tabnav].offpeak.min"
                   ng-click="updateOffPeakVal(scenario[tabnav].peak.min, tabnav);
                             updateRouteScorecard(scenario[tabnav].routeId, tabnav)"/>
         </div>
       </div>
-    </div>
-
+    
+      <h5> STATION TYPE </h5>
       <div class="btn-group btn-group-justified">
         <label  class="btn btn-info" btn-radio="0"
-                ng-model="scenario[tabnav].station"
+                ng-model="scenario[tabnav].station" 
                 ng-click="scenario[tabnav].station=0;
                           updateRouteScorecard(scenario[tabnav].routeId, tabnav)">
-          <i class="fa fa-star"></i>
-          <i class="fa fa-star-o"></i>
-          <i class="fa fa-star-o"></i>
+          <img class="stationIcons" src="public/imgs/pole_black_yellow.svg">
           Normal
         </label>
         <label  class="btn btn-info" btn-radio="1"
-                ng-model="scenario[tabnav].station"
+                ng-model="scenario[tabnav].station" 
                 ng-click="scenario[tabnav].station=1;
                           updateRouteScorecard(scenario[tabnav].routeId, tabnav)">
-          <i class="fa fa-star"></i>
-          <i class="fa fa-star"></i>
-          <i class="fa fa-star-o"></i>
+          <img class="stationIcons" src="public/imgs/platform_black_yellow.svg">
           Platform
         </label>
         <label  class="btn btn-info" btn-radio="2"
-                ng-model="scenario[tabnav].station"
+                ng-model="scenario[tabnav].station" 
                 ng-click="scenario[tabnav].station=2;
                           updateRouteScorecard(scenario[tabnav].routeId, tabnav)">
-          <i class="fa fa-star"></i>
-          <i class="fa fa-star"></i>
-          <i class="fa fa-star"></i>
+          <img class="stationIcons" src="public/imgs/brt_black_yellow.svg">
           BRT
         </label>
         <label class="btn btn-success" ng-click="saveAlt=true; createVar=false">
-          <i class="fa fa-cloud-upload"></i> Save Variation
+          <i class="fa fa-cloud-upload"></i> Save
         </label>
       </div>
     </div>
   </div>
 
 </div>
+<!--END of ROUTE EDITOR MODULE--> 
 
+<div class="controlModules">
 
 
-<div class="basecol">
+<!--POI MARKER ACTIVATION --> 
+
+<div class="basecol_POI col-md-3 "> 
+<!-- <h5 style="margin-bottom: 10px"> PLACES </h5> -->
+  <div class="fa-stack" ng-click="targetPOIUsers(); poiUserDots = !poiUserDots" style="margin-bottom: 8px">
+    <i class="fa fa-circle fa-stack-2x"></i>
+    <i class="fa fa-users fa-stack-1x fa-inverse destinationIcon iconContainer"></i>
```

Comment:
is the 8px margin necessary still for what you are doing? I was handling a specific edge case but I dont think it applies in vertical layout.